plugins/c9.cli.bridge/bridge_commands.js

```diff
@@ -66,6 +72,25 @@ define(function(require, exports, module) {
         }
         
         /***** Methods *****/
+        function createPipe(message, callback) {
+            tabManager.once("ready", function(){
+                tabManager.open( {
+                    focus: true, 
+                    editorType: "ace"
+                }, function(err, tab) {
+                    if (err) 
+                        return callback(err);
+                    callback(null, tab.name);
+                });
+            }); 
+        }
+        
+        function updatePipe(message, callback) {
+            tabManager.once("ready", function() {
+                tabManager.findTab(message.tab).document.value += message.data;    
```

Comment:
If the intention here is to see output of long running process, `document.value += ` will be slow, and generate large undo history, it's better to use editor.ace.session.insert
also findTab can be undefined if user closes the tab before the pipe is closed.